src/main/resources/db/changelog/changes/06-create-categories-table.yaml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-categories-table
+      author: Stepan
+      changes:
+        - createTable:
+            tableName: categories
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: description
+                  type: varchar(255)
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```

Comment:
each file should end with one empty line

Reply:
to avoid this
![image](https://github.com/Stipaxa-hub/store/assets/49767049/3ad7ec38-bb86-4aaa-a3c1-90d64eeed02d)